DimaDK24/orders-manager-frontend
# Feat: integrate redux

[Trello ticket](https://trello.com/c/k4jjda62)

# Problem statement

Currently app is just a markup. None of the buttons actually works. And I want to learn Redux with help of this project.

# Steps to test the changes

Open redux devtools
Click on "+" aka "add product" button
Click on "x" icon on product aka "remove product" button

Ensure app works as expected.
Ensure actions are visible in redux devtools.

# Solution description

Add several redux related packages:
- redux - redux itself
- react-redux - to nicely connect react to redux state
- redux-devtools-extension - to nicely add redux devtools

Create simple actions, reducer and store, connected to react.

# Sanity checklist


# Review checklist
